setup.rst

```diff
@@ -65,10 +65,10 @@ Symfony application using Composer:
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ composer create-project symfony/website-skeleton my_project_name
+    $ composer create-project symfony/website-skeleton:"^5.1" my_project_name
```

Comment:
This actually doesn't make a difference as `^5.1` allows 5.2 too. What about using `5.1.*` instead?

Reply:
Yes, `5.1.*` would be much more appropriate. thx to @wouterj for pushing fix to this. Totally my mistake, I was trying to use the same style of dependencies as in previous (versions of) guides but failed to check the result except for the fact that command is working